plugins/default/lib/disk_usage_mongo.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import argparse
+import re
+import sys
+import traceback
+from collections import Counter
+
+import nagios
+import openshift
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser(
+        description="Checks the disk usage (blocks and inodes)",
+    )
+    parser.add_argument(
+        "-w", "--warn", type=int, required=True,
+        help="set warning threshold of disk usage (%% of blocks or inodes)",
+    )
+    parser.add_argument(
+        "-c", "--crit", type=int, required=True,
+        help="set critical threshold of disk usage (%% of blocks or inodes), "
+             "must be higher than or equal the warning threshold",
+    )
+    parser.add_argument(
+        "-m", "--minimum", type=int, required=False, default=10,
+        help="set minimum threshold of disk usage  (%% of blocks or inodes), "
+             "details will only be reported for volumes over this value",
+    )
+    parser.add_argument(
+        "-p", "--project", required=False,
+        help="openshift project/namespace to use",
+    )
+    return parser
+
+
+check_disk_cmd = ("df", "--output=pcent,ipcent,target")
+# Example output:
+# Use% IUse% Mounted on
+#   1%    1% /
+check_disk_output_pattern = re.compile(r"\s*(\d+)%\s+(\d+)%\s+(?!\/etc\/hosts)(.+)$")
+
+
+def parse_df_line(line):
+    mo = check_disk_output_pattern.match(line)
+    if mo is None:
+        return ()
+    return mo.group(3), int(mo.group(1)), int(mo.group(2))
+
+
+def parse_df_lines(lines):
+    return filter(None, map(parse_df_line, lines.splitlines()))
+
+
+def analize(pod, container, disks, warning_threshold, critical_threshold):
```

Comment:
`analyze` ?

Reply:
Copy paste from another similar check :face_with_head_bandage:

Reply:
If the code is identical/ very similar - maybe it could re-used ? (unless there's a time-sensitive element to these changes of course)

Reply:
I think without some serious refactoring this is the quickest way to handle this particular requirement of targeting the particular mongo nodes. I'd like to get this into the 4.7.1 release.

Reply:
@davidkirwan I see that one of the linked issues has a fix version of 4.X-Future, and the other one doesn't have a fix version. I know @davidffrench did a lot of work grooming the backlog yesterday, so I'd suggest checking with him before merging anything into the FH-v4.7 branch for a 4.7.1.

Reply:
This one might have already been in progress before I got to it. Changed the fixversion to 4.7-Future. I don't see any reason not to include it in a patch release.